<html xmlns:stringutils="xalan://org.apache.tools.ant.util.StringUtils" xmlns:lxslt="http://xml.apache.org/xslt">
<head>
<META http-equiv="Content-Type" content="text/html; charset=UTF-8">
<title>Unit Test Results: All Tests</title>
<link rel="stylesheet" type="text/css" title="Style" href="stylesheet.css">
</head>
<body onload="open('allclasses-frame.html','classListFrame')">
<h1>Unit Test Results.</h1>
<table width="100%">
<tr>
<td align="left"></td><td align="right">Designed for use with <a href="http://www.junit.org/">JUnit</a> and <a href="http://ant.apache.org/">Ant</a>.</td>
</tr>
</table>
<hr size="1">
<h2>All Tests</h2>
<table class="details" border="0" cellpadding="5" cellspacing="2" width="95%">
<tr valign="top">
<th>Class</th><th>Name</th><th>Status</th><th width="80%">Type</th><th nowrap>Time(s)</th>
</tr>
<tr valign="top" class="TableRowColor">
<td><a href="com/ibm/ws/wssecurity/fat/cxf/0_FATSuite.html">FATSuite</a></td><td><a name="testCxfAsyncInvokeNonBlocking"></a><a href="com/ibm/ws/wssecurity/fat/cxf/0_FATSuite.html#testCxfAsyncInvokeNonBlocking">testCxfAsyncInvokeNonBlocking</a></td><td>Success</td><td></td><td>3.081</td>
</tr>
<tr valign="top" class="Failure">
<td><a href="com/ibm/ws/wssecurity/fat/cxf/0_FATSuite.html">FATSuite</a></td><td><a name="testCxfAsyncInvokeNonBlocking_EE8_FEATURES"></a><a href="com/ibm/ws/wssecurity/fat/cxf/0_FATSuite.html#testCxfAsyncInvokeNonBlocking_EE8_FEATURES">testCxfAsyncInvokeNonBlocking_EE8_FEATURES</a></td><td>Failure</td><td>2021-06-06-00:21:06:220 
Expected: 
Response: null
 But received: 
&lt;html&gt;&lt;head&gt;&lt;title&gt;CXF x509 Async Service Cleint&lt;/title&gt;&lt;/head&gt;&lt;body&gt;&lt;p&gt;Request: CXF x509 Async Service Client&lt;/p&gt;&lt;p&gt;Response: java.util.concurrent.ExecutionException: javax.xml.ws.soap.SOAPFaultException: com.ibm.ws.wssecurity.example.cbhwss4j.CommonPasswordCallbackWss4j cannot be found by com.ibm.ws.org.apache.cxf.rt.security.3.4.1_1.0.53.202106011250&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;
<br>
<br>
<code>junit.framework.AssertionFailedError: 2021-06-06-00:21:06:220 <br>Expected: <br>Response: null<br> But received: <br>&lt;html&gt;&lt;head&gt;&lt;title&gt;CXF x509 Async Service Cleint&lt;/title&gt;&lt;/head&gt;&lt;body&gt;&lt;p&gt;Request: CXF x509 Async Service Client&lt;/p&gt;&lt;p&gt;Response: java.util.concurrent.ExecutionException: javax.xml.ws.soap.SOAPFaultException: com.ibm.ws.wssecurity.example.cbhwss4j.CommonPasswordCallbackWss4j cannot be found by com.ibm.ws.org.apache.cxf.rt.security.3.4.1_1.0.53.202106011250&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;<br>
<br>	at com.ibm.ws.wssecurity.fat.utils.common.CommonTests.genericAsyncTestSub(CommonTests.java:629)<br>	at com.ibm.ws.wssecurity.fat.utils.common.CommonTests.genericAsyncTest(CommonTests.java:380)<br>	at com.ibm.ws.wssecurity.fat.utils.common.CommonTests.genericAsyncTest(CommonTests.java:327)<br>	at com.ibm.ws.wssecurity.fat.cxf.x509token.CxfX509ASyncTests.testCxfAsyncInvokeNonBlocking(CxfX509ASyncTests.java:120)<br>	at componenttest.custom.junit.runner.FATRunner$1.evaluate(FATRunner.java:197)<br>	at componenttest.custom.junit.runner.FATRunner$2.evaluate(FATRunner.java:318)<br>	at componenttest.custom.junit.runner.FATRunner.run(FATRunner.java:171)<br>	at componenttest.rules.repeater.RepeatTests$CompositeRepeatTestActionStatement.evaluate(RepeatTests.java:115)<br>
</code></td><td>11.555</td>
</tr>
</table>
</body>
</html>
